saleor/plugins/manager.py

```diff
@@ -885,6 +885,15 @@ def fulfillment_metadata_updated(self, fulfillment: "Fulfillment"):
             "fulfillment_metadata_updated", default_value, fulfillment
         )
 
+    def fulfillment_tracking_number_updated(self, fulfillment: "Fulfillment"):
```

Comment:
It seems like we do have manager function for this action already: `tracking_number_updated`. I guess we can use it for this purpose and just override it in the WebhookPlugin, instead of adding a new hook `fulfillment_tracking_number_updated`.